lib/govuk/dummy_content_store/content.rb

```diff
@@ -15,6 +16,13 @@ def call(env)
         example = repository.find_by_base_path(env["PATH_INFO"])
         if example
           present_example(example)
+        elsif fallback_url ||= "https://www.gov.uk/api/content"
```

Comment:
But if I configure it to set DUMMY_CONTENT_STORE_FALLBACK_URL, they can be used both I think.